core/src/main/scala/cats/Traverse1.scala

```diff
@@ -0,0 +1,133 @@
+package cats
+
+import cats.data.NonEmptyList
+import simulacrum.typeclass
+
+@typeclass trait Traverse1[F[_]] extends Traverse[F] with Reducible[F] {
+
+  def traverse1[G[_]: Apply, A, B](fa: F[A])(f: A => G[B]): G[F[B]]
+
+  def sequence1[G[_]: Apply, A](fga: F[G[A]]): G[F[A]] =
+    traverse1(fga)(identity)
+
+  def flatTraverse1[G[_], A, B](fa: F[A])(f: A => G[F[B]])(implicit G: Apply[G], F: FlatMap[F]): G[F[B]] =
+    G.map(traverse1(fa)(f))(F.flatten)
```

Comment:
Ditto for this, and `flatSequence`, `sequence`, `traverse1U`, `sequence1U` in the same class.